### ISSUE TITLE
MSI unatteded setup problem

### ISSUE THREAD

**Issue by rwalke** [NetLogo/NetLogo#1517]
I have tried to prepare an unattended setup using NetLogo-6.0.2-64.msi .
We use Windows 7 (64-bit).
The installation does not show any error, but nothing happens.

Debugging shows:
"MSI (s) (8C:CC) [10:26:51:097]: Windows Installer reconfigured the product. Product Name: NetLogo 6.0.2. Product Version: 6.0.2. Product Language: 1033. Manufacturer: CCL. Reconfiguration success or error status: 0."

It seems to me, that if the installation runs in SYSTEM context and not in a user context, it causes problems.

Further trials:
ALLUSERS=1 ADDLOCAL=BaseInstall
NetLogo-6.0.2-32.msi
NetLogo-6.0.1-64.msi
NetLogo-6.0-64.msi
NetLogo-5.3.1-64.msi

Any hints or ideas are welcome.
Best,
Rainer

**Comment by mrerrormessage** [comment]
Thanks for posting this! I just verified that I am able to install NetLogo 6.0.2-64 on a Windows 7 (64-bit Enterprise edition) virtual machine. Here were the steps I used:

* Downloaded installer
* Opened Powershell as administrator
* Ran the following command `msiexec /i C:\Path\to\NetLogo-6.0.2-64.msi /qn /log logfile.txt ALLUSERS=1 ADDLOCAL=BaseInstall`
* The command returned right away, but it took a bit (maybe 15-30s) to actually see that NetLogo had installed.

I notice in our Advanced Installation page we don't have the `/qn` flag passed to `msiexec`, so the first time I ran `msiexec` I saw the installation dialog pop up (which would obviously botch truly headless installation). I don't think that flag is necessary in Windows 10, but it would probably be worth adding in general.

Are you able to get NetLogo to install using the above steps? If so, is there other information missing from the advanced installation page which would clear things up?
